its/src/test/java/com/sonar/it/scanner/msbuild/sonarcloud/CloudTests.java

```diff
@@ -51,4 +58,12 @@ private void analyzeEmptyProject() {
     AnalysisContext.forCloud("Empty").runAnalysis();
     ContextExtension.cleanup();
   }
+
+  private void ensureEnvironment(String envVariable, String envVariableValue)
```

Comment:
This needs the "Name" to really distinguish what it means. As the current naming does not describe the difference between those two
```suggestion
  private void ensureEnvironment(String envVariableName, String envVariableValue)
```
or
```suggestion
  private void ensureEnvironment(String name, String value)
```